modules/openapi-generator/src/main/resources/aspnetcore/2.0/controller_impl.mustache

```diff
@@ -0,0 +1,50 @@
+{{#usePartialControllerClass}}
+{{>partial_header}}
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;{{#useSwashbuckle}}
+using Swashbuckle.AspNetCore.SwaggerGen;{{/useSwashbuckle}}
+using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+using {{packageName}}.Attributes;
+using {{packageName}}.Models;
+
+namespace {{packageName}}.Controllers
+{ {{#operations}}
+    /// <summary>
+    /// {{description}}
+    /// </summary>{{#description}}
+    [Description("{{description}}")]{{/description}}
+    public partial class {{classname}}Controller : Controller
+    { {{#operation}}
+        /// <summary>
+        /// {{#summary}}{{summary}}{{/summary}}
+        /// </summary>{{#notes}}
+        /// <remarks>{{notes}}</remarks>{{/notes}}{{#allParams}}
+        /// <param name="{{paramName}}">{{description}}</param>{{/allParams}}
+        /// <param name="result_">Operation result</parm>
+        partial void {{operationId}}_impl({{#allParams}}{{&dataType}} {{paramName}}, {{/allParams}}ref IActionResult result_)
+        {
+{{#responses}}{{#dataType}}
+            //TODO: Uncomment the next line to return response {{code}} or use other options such as result_ = this.NotFound(), result_ = this.BadRequest(..), ...
+            // result_ = StatusCode({{code}}, default({{&dataType}})); return;
+{{/dataType}}{{^dataType}}
+            //TODO: Uncomment the next line to return response {{code}} or use other options such as result_ = this.NotFound(), result_ = this.BadRequest(..), ...
+            // result_ = StatusCode({{code}}); return;
+{{/dataType}}{{/responses}}
+{{#returnType}}
```

Comment:
Minor suggestion: Using https://github.com/samskivert/jmustache#newline-trimming to more easily control new line and spacing.